js_modules/dagster-ui/packages/ui-core/src/assets/asset-checks/AssetCheckDetailModal.tsx

```diff
@@ -285,7 +285,7 @@ export function MigrationRequired() {
         description={
           <Box flex={{direction: 'column'}}>
             <Body2 color={Colors.Gray700} style={{padding: '6px 0'}}>
-              A data migration is required to use asset checks. Run{' '}
+              A database schema migration is required to use asset checks. Run{' '}
```

Comment:
Consider linking docs as well

Reply:
I think we need to add a docs page about instance migration. We have the k8s one but not a generic one afaik